curvefs/src/client/common/common.h

```diff
@@ -64,6 +65,26 @@ std::ostream &operator<<(std::ostream &os, MetaServerOpType optype);
 
 const uint32_t MAXXATTRLENGTH = 256;
 
+const char kCurveFsWarmupXAttr[] = "curvefs.warmup.op";
+const char kCurveFsWarmupOpAdd[] = "add";
```

Comment:
should you add prefix `curvefs.warmup.add`?

Reply:
> should you add prefix `curvefs.warmup.add`?

no，add means warmup operation，will add del/query in feature